client/src/pages/ManageVocab/ManageVocab.js

```diff
@@ -0,0 +1,135 @@
+import React, { useEffect, useState } from 'react'
+import PropTypes from 'prop-types'
+import ManageView from 'components/ManageView'
+
+import { useErrorWrap } from 'hooks'
+import { useSelector, useDispatch } from 'react-redux'
+import { updateVocabs, setField, updateNumVocab } from 'slices/language.slice'
+import { updateVocabItems } from 'api'
+import _ from 'lodash'
+import { INDICATOR_TYPES } from 'utils/constants'
+
+const ManageVocab = ({ navigation }) => {
+  const errorWrap = useErrorWrap()
+  const dispatch = useDispatch()
+  const { lessonData, currentCourseId, currentLessonId } = useSelector(
+    (state) => state.language,
+  )
+  //   const allVocab = lessonData.vocab
+  const [selected, setSelected] = useState([])
+  const [unselected, setUnselected] = useState([])
+
+  /**
+   * Filters all of the vocab items into selected and unselected lists
+   */
+  useEffect(() => {
+    let selectedList = []
+    let unselectedList = []
+    for (let i = 0; i < lessonData.vocab.length; i += 1) {
+      const item = lessonData.vocab[i]
+
+      const formattedItem = {
+        _id: item._id,
+        title: item.original,
+        body: item.translation,
+        indicatorType: INDICATOR_TYPES.NONE, // TODO: remove hard-coded value
```

Comment:
I think we can remove the TODO. We had indicators in last semester's designs, but we've moved away from them this year.